WordPress/WordPressApi/WordPressComRestApi.swift

```diff
@@ -227,6 +230,7 @@ final class WordPressComRestAPIResponseSerializer: AFJSONResponseSerializer
         let extraStatusCodes = NSMutableIndexSet(indexSet: self.acceptableStatusCodes!)
         extraStatusCodes.addIndexesInRange(NSRange(400...500))
         self.acceptableStatusCodes = extraStatusCodes
+        self.readingOptions = [.AllowFragments]
```

Comment:
I see why you added this, but I think this opens the door to more problems down the line. The `isEmailAvailable` case is a very specific edge case that should eventually be fixed in the backend. But with this, you're changing the behavior for every request. I think when we parse JSON in the app we kind of assume that we're getting a dictionary, and relying on it being an error otherwise. I'd rather keep the previous workaround and not change the default `readingOptions`.